servicetalk-capacity-limiter-api/src/main/java/io/servicetalk/capacity/limiter/api/RequestDroppedException.java

```diff
@@ -18,24 +18,25 @@
 import javax.annotation.Nullable;
 
 /**
- * An {@link Exception} to indicate that a request was rejected by a client/server due to capacity constraints.
+ * An {@link Exception} to indicate that a request was dropped by a client/server likely due to capacity constraints or
+ * as-such interpretation of peer feedback according to configuration.
  */
-public class RequestRejectedException extends RuntimeException {
+public class RequestDroppedException extends RuntimeException {
```

Comment:
Any thoughts on "dropped" vs "throttled"?

Reply:
Thought so too, but i prefer the dropped, it aligns with other `Ticket` APIs and also, there are both Capacity & Breaker use-cases that result on this. So "throttled" sounds more related to capacity only.